src/App.tsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 
 import { Header } from './components/header/Header';
+// import { ProductCard } from './components/productCard/productCard';
 import { Footer } from './components/footer/Footer';
 import './App.scss';
 
 function App() {
   return (
     <div className="App">
       <Header />
+      {/* <ProductCard /> */}
```

Comment:
For the future don't comment it, ur component will be used in other places. Just don't use it here